---
layout: default
title: Kevin C Lee - CV
---
<div class="blurb">
	<h1>Curriculum Vitae</h1>
	<h2>Kevin Chien-Yu Lee, PhD</h2>
cykevinlee (at) gmail (dot) com
<h3>Education_________________________________________________________________________</h3>

2010-2015: PhD., School of Science/ Extremophile Research Group
University of Waikato in collaboration with GNS Science, New Zealand

*  PhD has been approved, currently awaiting graduation.
*  Advisors: Ian R. McDonald, Matthew B. Stott, Xochitl C. Morgan
*  Thesis: “The First Insights into the Phylogeny, Genomics, and Ecology of the Novel Bacterial Phylum Armatimonadetes”
*  Microbiology, Molecular Ecology, and Comparative Genomics

2007 - 2009: Master of Science with first class honours
University of Waikato, New Zealand
*  Thesis: “The Physiological and Ecological Characterisation of the First Cultivated Species of the Candidate Division OP10”

2004 - 2008: Bachelor of Science (Technology) 
University of Waikato, New Zealand
* Microbiology, Biochemistry, and Genetics

<h3>Scientific Experience__________________________________________________________</h3>

2015-Current: Institute for Applied Ecology New Zealand, AUT University, Auckland, New Zealand
* Postdoctral research fellow
2009-2015: Extremophile Research Group, GNS Science, Wairakei, New Zealand
* Conducted phylogenetic analyses to determine the boundaries of the novel bacterial phylum Armatimonadetes and closely-associated candidate division. Correlated the resulting clades with established taxonomic designations (Dunfield et al., AEM, 2012; Lee et al., AEM, 2013)
* Assembled genome of Chthonomonas calidirosea, utilised primer-walking techniques to improve the assembly, and conducted genome analysis (Lee et al. ISMEJ, 2014)
* Cultured C. calidirosea and conducted sugar-utilisation experiments with HPLC analysis
* Extracted DNA from several recalcitrant C. calidirosea strains, performed mapping assemblies, and conducted a comparative genomic study (manuscript in revision)
* Analysis of sequencing data from multiple platforms (Illumina, Ion Torrent, 454, and Sanger)
* Processed and analysed sequencing data (16S rRNA genes and whole genome) and interacted with data archives (NCBI GenBank & EMBL-EBI) for data curation and submission
* De novo & hybrid assembly (454 and Illumina) and characterisation of a novel group 4 thermophilic Acidobacteria sp. genome (manuscript in preparation)

2008-2009: Extremophile Research Group, GNS Science, Wairakei, New Zealand
* Physiological characterisation of the novel species C. calidirosea, including a wide range of substrate utilisation and temperature/pH growth ranges (Lee, MSc. thesis, 2009, Lee et al., IJSEM, 2011)
* Designed specialised PCR primers to assist in the detection and isolation of the candidate division OP10/Armatimonadetes in environmental samples and mixed cultures.

2004-2008: Internship programme and employment during undergraduate study
* SAITL Dairy Laboratory - Industrial milk microbiology testing for antibiotic residue and microbial contaminants.
* Honey Research Unit, University of Waikato - Analysis for honey purity using a viscometer, and honey-Staphlococcus aureus inhibition study with tetrazolium spectroscopic assay.
* New Zealand Forest Research Institute Ltd. - Conducted qualitative reverse-transcription PCR to assess the expression of genetically-modified eukaryotic cell lines.
* Diagnostic Medlab - Sample processing using microbiological techniques for the medical laboratory.

<h3>Software Experience_______________________________________________________________</h3>
*	Phylogenetic analysis – ARB, SILVA database, MrBayes, RAxML, PhyML, Phylip, PhyloPhlAn
*	(Meta)genome assembly & analysis – MIRA & SPAdes assemblers, bedtools, samtools, Artemis, Staden package, Bowtie2, Mauve
*	Microbial community analysis: QIIME, Mothur, USEARCH, Primer 6
*	Linux, R (e.g., vegan, phyloseq, ggplot2), Python

<h3>Publications______________________________________________________________________</h3>
•	Archer, S.D.J., de los Ríos, Lee, K.C., Niederberger, T.S., Cary, S.C., Coyne, K.J., Douglas, S., Lacap-Bugler, D.C., Pointing, S.B. (Submitted) A., Endolithic microbial diversity in sandstone and granite from the McMurdo Dry Valleys, Antarctica. Polar Biology
•	Lee, K. C., Stott, M. B., Dunfield, P. F., Huttenhower, C., McDonald, I. R., & Morgan, X.C. (In revision). The Chthonomonas calidirosea genome is highly conserved across geography and distinct chemical and microbial environments in the Taupō Volcanic Zone. Applied and Environmental Microbiology
•	Lee, K. C., Morgan, X. C, Power, J. F., Dunfield, P. F., Huttenhower, C., & Stott, M. B. (2015). Complete genome sequence of the thermophilic Acidobacteria, Pyrinomonas methylaliphatogenes type strain K22T. Standards in Genomics Sciences. 10, 101.
•	Lee, K. C., Morgan, X. C., Dunfield, P. F., Tamas, I., McDonald, I. R., & Stott, M. B. (2014). Genomic analysis of Chthonomonas calidirosea, the first sequenced isolate of the phylum Armatimonadetes. The ISME Journal, 1–12
•	Lee, K. C.-Y., Stott, M. B., & Dunfield, P. F. (2014). Phylum Armatimonadetes. In E. Rosenberg, E. F. DeLong, F. Thompson, S. Lory, & E. Stackebrandt (Eds.), The Prokaryotes (4th ed). Springer
•	Lee, K. C. Y., Herbold, C. W., Dunfield, P. F., Morgan, X. C., McDonald, I. R., & Stott, M. B. (2013). Phylogenetic delineation of the novel phylum Armatimonadetes (former candidate division OP10) and definition of two novel candidate divisions. Applied and environmental microbiology, 79(7), 2484–7
•	Dunfield, P. F., Tamas, I., Lee, K. C., Morgan, X. C., McDonald, I. R., & Stott, M. B. (2012). Electing a candidate: a speculative history of the bacterial phylum OP10. Environmental microbiology, 14(12), 3069–80
•	Vyssotski, M., Lee, K. C.-Y., Lagutin, K., Ryan, J., Morgan, X. C., & Stott, M. B. (2011). Fatty Acids of Chthonomonas calidirosea, of a novel class Chthonomonadetes from a recently described phylum Armatimonadetes. Lipids, 46(12), 1155–61
•	Lee, K. C.-Y., Dunfield, P. F., Morgan, X. C., Crowe, M. A., Houghton, K. M., Vyssotski, M., … Stott, M. B. (2011). Chthonomonas calidirosea gen. nov., sp. nov., an aerobic, pigmented, thermophilic micro-organism of a novel bacterial class, Chthonomonadetes classis nov., of the newly described phylum Armatimonadetes originally designated candidate division OP10. International Journal of Systematic and Evolutionary Microbiology, 61(Pt 10), 2482–90. 

<h3>Awards___________________________________________________________________________</h3>
* 2010: Sarah Beanland Memorial Ph.D. Fellowship
* 2009: University of Waikato Masters Research Scholarship
* 2009: GNS Science Masters Research Study Award
* 2004: Tearaway Waikato University Scholarships

<h3>Selected Presentations____________________________________________________________</h3>
* New Zealand Microbiological Society 58th Annual Conference, University of Waikato. Nov 2013. The genomics and ecology of the extremophilic scavenger Chthonomonas calidirosea
* Huttenhower Lab, Department of Biostatistics, Harvard School of Public Health. Jul 2013. The genomic and ecological characterisation of Chthonomonas calidirosea T49
* New Zealand Microbial Ecology Consortium, University of Auckland. Jan 2012. Ecological implications of the genomic and phenotypic characteristics of an Armatimonadetes strain

<h3>Poster Session Presentations______________________________________________________</h3>
* Lee, K.C., Morgan, X.C., Ryan, J., Lagutin, K., Vyssotski, M., McDonald, I.R., Stott, M.B. Physiological characterization and genome sequence of the first cultured representative of candidate division OP10, ‘Khthonomonas calidirosea’. 13th International Symposium on Microbial Ecology (Seattle, USA, 22-27 Aug 2010)
* Lee, C.K., Morgan, X.C., Ryan, J., Lagutin, K., Vyssotskii, M. McDonald, I.R. and Stott, M.B. The physiological and ecological characterisation of the first cultivated species of candidate division OP10. COMBIO conference (Christchurch, New Zealand, 6-10 Dec 2009)

<h3>Professional Activities___________________________________________________________</h3>
* Member of New Zealand Microbiological Society (NZMS)
* Member of New Zealand Microbial Ecology Consortium (NZMEC)
* Member of American Society for Microbiology (ASM)

<h3>Languages_________________________________________________________________________</h3>
* Spoken: English (fluent), Mandarin Chinese (native)

References____________________________________________________________________________________________________________________________________________________

Available upon request.
	
</div><!-- /.blurb -->
